src/react/backend/location/__tests__/LocationEditor.test.jsx

```diff
@@ -12,12 +12,13 @@ describe('LocationEditor', () => {
         selectInput.simulate('keyDown', { key: 'ArrowDown', keyCode: 40 });
 
         const options = component.find('div.sc-select__option');
-        expect(options.length).toBe(6);
+        expect(options.length).toBe(7);
         expect(options.first().find('span').text()).toBe('Storage Service for ARTESCA');
         expect(options.at(1).find('span').text()).toBe('Scality RING with S3 Connector');
-        expect(options.at(2).find('span').text()).toBe('Amazon S3');
-        expect(options.at(3).find('span').text()).toBe('Google Cloud Storage');
-        expect(options.at(4).find('span').text()).toBe('Microsoft Azure Blob Storage');
-        expect(options.at(5).find('span').text()).toBe('Local Filesystem');
+        expect(options.at(2).find('span').text()).toBe('Scality Artesca S3');
+        expect(options.at(3).find('span').text()).toBe('Amazon S3');
+        expect(options.at(4).find('span').text()).toBe('Google Cloud Storage');
+        expect(options.at(5).find('span').text()).toBe('Microsoft Azure Blob Storage');
+        expect(options.at(6).find('span').text()).toBe('Local Filesystem');
```

Comment:
Not a fan of index based tests, could this be changed?

Reply:
I agree on the indexing but I felt it would be a bit much to restructure, but it did seem this order was important as it matched the other projects. Could not tell you if that is true though, I just added Artesca after Ring as it made sense to me.

Reply:
@nicolas2bert scality types first?

Reply:
Checked with Thomas that Artesca should be before Ring, changing now.